cmd/wallet-web/src/pages/OIDCShare.vue

```diff
@@ -22,7 +22,7 @@
     <WACI-error v-else-if="errors.length" @click="cancel" />
 
     <!-- Credentials Missing State -->
-    <WACI-credentials-missing v-else-if="showCredentialsMissing" @click="cancel" />
+    <WACI-credentials-missing v-else-if="noCredentialFound" @click="cancel" />
```

Comment:
These changes are already merged into `main`, they should not come up in your PR. You should probably rename your branch from `main` to `issue-1563`. Then you'd be able to perform the rebasing properly

Reply:
Ok I will rebase and create a new PR

Reply:
No worries 👍🏼